mono/metadata/debug-mono-ppdb.c

```diff
@@ -436,7 +435,8 @@ mono_ppdb_lookup_location (MonoDebugMethodInfo *minfo, uint32_t offset)
 	}
 
 	location = g_new0 (MonoDebugSourceLocation, 1);
-	location->source_file = docname;
+	if (docname != NULL && docname[0] != 0)
```

Comment:
cosmetic: `if (docname && docname [0])`

Reply:
or `if (!docname || docname [0])`
but yours is clearer.

Reply:
done!